web/src/api/fbaAPI.ts

```diff
@@ -96,6 +103,12 @@ export interface FuelType {
   description: string
 }
 
+export interface FireCentreHfiFuelsData {
```

Comment:
I'm not sure this interface accurately reflects the actual shape of the object? This interface makes it look like a fire centre name will only have one property, it's not clear that there could be multiple properties with type `[fire_zone_id: number]: FireZoneThresholdFuelTypeArea[]`.  I might be old school, but I'd prefer to see an array of `[fire_zone_id: number]: FireZoneThresholdFuelTypeArea[]`. Perhaps @conbrad has a different opinion?

Reply:
I agree, I think the endpoint that existed had a lot of rough edges, I can address them in the critical hours PR.

Reply:
On second thought, I think this is ok, I'm more used to seeing this syntax in a Type definition. I think it's fine (and a matter of preference).

Reply:
Nice catch, I agree, I'll start doing some refactors once the tabs PR is in.

Reply:
Oops, I added this comment to the wrong PR.
 Index signatures are totally valid, it's the rigid nature of interfaces from my C# background coming back to haunt me!